lib/charms/data_platform_libs/v0/data_interfaces.py

```diff
@@ -347,6 +351,12 @@ def _on_karapace_subject_allowed(self, event: SubjectAllowedEvent):
         tls = event.tls
         endpoints = event.endpoints
         ...
+
+    def _on_karapace_entity_created(self, event: SubjectEntityCreatedEvent):
```

Comment:
Nit: Following the naming of other Data Platform products, this should be:

```suggestion
    def _on_subject_entity_created(self, event: SubjectEntityCreatedEvent):
```